docs/docs/install-homebrew.mdx

```diff
@@ -1,17 +1,25 @@
 A [Homebrew][brew] formula is available for easy installation. When installing Homebrew for Linux, be sure to follow *[Next steps][nextsteps]* instructions to add Homebrew to your PATH and to your bash shell profile script.
 
+There are two different ways to install Oh My Posh via Homebrew:
+
+- **#A**
+
 ```bash
-brew tap jandedobbeleer/oh-my-posh
-brew install oh-my-posh
+brew install jandedobbeleer/oh-my-posh/oh-my-posh
```

Comment:
I would only add this line, 2 ways of installing where one is one that I don't maintain is not something I wish to support ;-). The fact that it also installs golang is irrelevant in my book as that's how brew works. That is completely unrelated to oh-my-posh and goes for any installation.

Reply:
@JanDeDobbeleer Got your thought. Removed others while the executable path should be `$(brew --prefix)/bin` indeed.